src/site/resources/index.html

```diff
@@ -0,0 +1,160 @@
+				StackOverflow
+                            </a>
+                        </li>
+                        <li>
+                            <a href="./project-info.html" title="Java package manager" class="ext-link">
+                                 <img src="img/icon-link-package-manager.svg" alt="Package Manager icon" />
+                                Package Information
+                            </a>
+                        </li>
+                    </ul>
+                </div><!-- end of .container -->
+            </section><!-- end of .featuring -->
+
+            <section class="block about">
+                <div class="container clearfix">
+                  <h3 class="block-title">What is it?</h3>
+
+                  <p><code>gcloud</code> is a client library for accessing Google
+		    Cloud Platform services that significantly reduces the boilerplate
+		    code you have to write. The library provides high-level API
+		    abstractions so they're easier to understand. It embraces
+		    idioms of Java, works well with the standard library, and
+		    integrates better with your codebase.
+		    All this means you spend more time creating code that matters
+		    to you.</p>
+		  
+		  <p><code>gcloud</code> is configured to access Google Cloud Platform
+		    services and authorize (OAuth 2.0) automatically on your behalf.
+		    Add the <code>gcloud</code> dependency to your project and get a private key to be
+		    up and ready to go. Better yet, if you are running on a Google
+		    Compute Engine instance, the private key is automatically detected.
+		  
+                  <h4>Retrieve Datastore Entries</h4>
+		  
+                  <div hljs="" language="js">
+		    <pre><code class="hljs java"><span class="hljs-keyword">import</span> com.google.gcloud.datastore.Datastore;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.DatastoreFactory;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.DatastoreOptions;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.Entity;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.Key;
+<span class="hljs-keyword">import</span> com.google.gcloud.datastore.KeyFactory;
+
+DatastoreOptions options = DatastoreOptions.builder().projectId(<span class="hljs-string">PROJECT_ID</span>).build();
+Datastore datastore = DatastoreFactory.instance().get(options);
+KeyFactory keyFactory = datastore.newKeyFactory().kind(<span class="hljs-string">KIND</span>);
+Key key = keyFactory.newKey(keyName);
+Entity entity = datastore.get(key);</code></pre></div>
+                </div><!-- end of .container -->
+            </section><!-- end of .featuring -->
+
+            <section class="block faq">
+                <div class="container">
+                    <h3 class="block-title">FAQ</h3>
+		    <h4>What is the relationship between the gcloud package and the gcloud command-line tool?</h4>
```

Comment:
I think for Java saying "gcloud package" is not as clear as in Python or node where "gcloud" is indeed the package name...